pubnub-gson/pubnub-gson-api/src/main/java/com/pubnub/api/java/util/TimetokenUtil.java

```diff
@@ -0,0 +1,95 @@
+package com.pubnub.api.java.util;
+
+import java.time.Instant;
+
+public class TimetokenUtil {
```

Comment:
nit: even if we're writing a separate implementation, this class could have been written in Kotlin, we don't need to write Java code in `pubnub-gson`

Reply:
I like this suggestion! Thanks. I will rewrite it.